### PR TITLE
Bug 1144453 - [RTL][Contacts] In contact edit screen the selected dat…

### PR DESCRIPTION
**Pull request by ferjm**
…e is truncated making it difficult to determine which month is selected. r=francisco

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=097ce7b28a55cd71ef99f1987d098f565fdc7487

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1ac9dd445484b5fef02473959356082a8f2581d3

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5e4cbba627ae3e5b0d3127afefb122654cc2f109

File: shared/js/contacts/import/utilities/misc.js [hunk 1]
```diff
@@ -42,9 +42,9 @@ if (!utils.misc) {
 
       var year = normalizedDate.getFullYear();
       if (year !== FLAG_YEAR_IGNORED) {
-        options.year = 'numeric';
+        format.year = 'numeric';
```
Comment (arcturus) [inline shared/js/contacts/import/utilities/misc.js]: This could bypass the selected format isnt?
Not a blocker right now, so we can live with it.

**Pull request merged**